frontend/src/components/Account/MemberAccount/MemberAccount.scss

```diff
@@ -0,0 +1,9 @@
+.member-container {
+    p {
+        font-size: 14px;
+    }
+    .member-select {
```

Comment:
Are you going for a `.a__b` or for a `.a-b` approach? We should be consistent both within our own code as well as the codebase itself.

Reply:
fixed to a__b